src/app/shadow.ts

```diff
@@ -0,0 +1,7 @@
+export type Shadow = {
```

Comment:
export type Shadow = {
   [key in 'horizontalShadow' | 'verticalShadow' | 'spread' | 'blur' ] : number,
  color: string
}